R/dummy_multi_choice.R

```diff
@@ -1,9 +1,8 @@
 #' Handle levels in multiple predictors together
 #'
-#' `step_dummy_multi_choice()` creates a *specification* of a recipe
-#'  step that will convert multiple nominal data (e.g. character or factors)
-#'  into one or more numeric binary model terms for the levels of
-#'  the original data.
+#' `step_dummy_multi_choice()` creates a *specification* of a recipe step that
+#' will convert multiple nominal data (e.g. character or factors) into one or
```

Comment:
```suggestion
#' will convert multiple nominal data (e.g. characters or factors) into one or
```

Reply:
What does the "multiple" refer to here? Columns?

Reply:
columns